arrabiata/src/lib.rs

```diff
@@ -0,0 +1,73 @@
+use strum::EnumCount as _;
+
+pub mod column_env;
+pub mod columns;
+pub mod constraints;
+pub mod interpreter;
+pub mod logup;
+pub mod poseidon_3_60_0_5_5_fp;
+pub mod poseidon_3_60_0_5_5_fq;
+pub mod proof;
+pub mod prover;
+pub mod verifier;
+pub mod witness;
+
+/// The maximum degree of the polynomial that can be represented by the
+/// polynomial-time function the library supports.
+pub const MAX_DEGREE: u64 = 5;
+
+/// The minimum SRS size required to use Nova, in base 2.
+/// Requiring at least 2^16 to perform 16bits range checks.
```

Comment:
Is this assuming we use plookup ?

Reply:
Plookup or logup. But no lookup argument is implemented for now. There was an initial plan to use one, but I would revisit this later, after the PoC.